### ISSUE TITLE
Auto-casting dates results in double quotes

### ISSUE THREAD

**Issue by rosmo** [arangodb/aqbjs#8]
index.js has this:
```javascript
    if (obj instanceof Date) {
      return QB(JSON.stringify(obj));
    }
```
Thus QB(new Date()); results in:
```javascript
{ value: '"2015-02-19T20:40:52.117Z"' }
```

**Comment by pluma** [comment]
Yupp, should be autoCastToken, not QB.

**Comment by rosmo** [comment]
Thanks!

**Comment by pluma** [comment]
No problem. Thanks for reporting.

**Comment by rosmo** [comment]
Do you think you could do a minor release with this fix? I'm about to push a new project which requires this fix.

**Comment by pluma** [comment]
The fix is already published in 1.8.3.

**Comment by rosmo** [comment]
Good, thanks again :100:
